global.html

```diff
@@ -99,6 +103,9 @@
         };
         xhr.send();
       }
+
+      safari.extension.addContentScriptFromURL(safari.extension.baseURI + 'inject.js', ['safari-reader://*/*'], null, true);
+      safari.extension.addContentStyleSheetFromURL(safari.extension.baseURI + 'style.css', ['safari-reader://*/*'], null, true);
```

Comment:
According to the [Safari Extensions Development Guide](https://developer.apple.com/library/safari/documentation/Tools/Conceptual/SafariExtensionGuide/WorkingWiththeReader/WorkingWiththeReader.html#//apple_ref/doc/uid/TP40009977-CH22-SW6) this should not have to be done programatically and can be done through a whitelist setting in the Extension Builder. I couldn’t get this to work though.